src/accessibility/videos-and-recordings/index.md.njk

```diff
@@ -0,0 +1,17 @@
+---
+title: Videos and recordings
+description: Videos and recordings
+section: Accessibility
+theme: Resources
+layout: layout-pane.njk
+order: 13
+---
+Before you can publish a recording on Stream or SharePoint, you must make sure it is accessible. 
+This means providing an accurate transcript.
+
+Although Microsoft creates an automatic [transcript](https://support.microsoft.com/en-gb/office/view-live-transcription-in-a-teams-meeting-dc1a8f23-2e20-4684-885e-2152e06a4a8b) and subtitles it will always have errors. 
```

Comment:
```suggestion
Although [Microsoft Teams creates an automatic transcript](https://support.microsoft.com/en-gb/office/view-live-transcription-in-a-teams-meeting-dc1a8f23-2e20-4684-885e-2152e06a4a8b) and subtitles it will always have errors.
```